src/CSET/operators/plot.py

```diff
@@ -458,8 +458,10 @@ def _plot_and_save_postage_stamp_spatial_plot(
 
     # Put the shared colorbar in its own axes.
     colorbar_axes = fig.add_axes([0.15, 0.07, 0.7, 0.03])
-    colorbar = fig.colorbar(plot, colorbar_axes, orientation="horizontal")
-    colorbar.set_label(f"{cube.name()} / {cube.units}")
+    colorbar = fig.colorbar(
+        plot, colorbar_axes, orientation="horizontal", pad=0.042, shrink=0.7
+    )
+    colorbar.set_label(f"{cube.name()} / {cube.units}", size=16)
```

Comment:
```suggestion
    colorbar.set_label(f"{cube.name()} ({cube.units})", size=16)
```
Noticed that the units are displayed inconsistently.